security: TestRotateCerts failed

security.TestRotateCerts [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/20873485?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_TestsAwsLinuxArm64_UnitTests/20873485?buildTab=artifacts#/) on master @ [e29fd33d0424e1c86f5ff4c1f6a5e7d25f0819b3](https://github.com/cockroachdb/cockroach/commits/e29fd33d0424e1c86f5ff4c1f6a5e7d25f0819b3):


```
=== RUN   TestRotateCerts
    test_log_scope.go:171: test logs captured to: /artifacts/tmp/_tmp/7e9178a8651a0c2cb646d0ebdb5ed75e/logTestRotateCerts3357829972
    test_server_shim.go:94: cluster virtualization disabled due to issue: #110007 (expected label: C-bug)
    test_server_shim.go:668: DRPC is enabled (via override by TestingGlobalDRPCOption)
    certs_rotation_test.go:199: issuing SIGHUP
    certs_rotation_test.go:314: issuing SIGHUP
    certs_rotation_test.go:320: condition failed to evaluate within 45s: from certs_rotation_test.go:88: third HTTP client failed: Expected OK, got "500 Internal Server Error" with body: {
          "error": "connection closed",
          "code": 2,
          "message": "connection closed",
          "details": [
          ]
        }
    panic.go:615: -- test log scope end --
test logs left over in: /artifacts/tmp/_tmp/7e9178a8651a0c2cb646d0ebdb5ed75e/logTestRotateCerts3357829972
--- FAIL: TestRotateCerts (48.23s)
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/security-engineering @cockroachdb/product-security
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestRotateCerts.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-57970

--- a/pkg/security/certs_rotation_test.go
+++ b/pkg/security/certs_rotation_test.go
@@ -66,6 +66,9 @@ func TestRotateCerts(t *testing.T) {
 		InsecureWebAccess: true,
 
 		DefaultTestTenant: base.TestIsForStuffThatShouldWorkWithSecondaryTenantsButDoesntYet(110007),
+		// Disable DRPC for this test due to #159661: since it's flaky
+		// with "connection closed" errors when DRPC is enabled.
+		DefaultDRPCOption: base.TestDRPCDisabled,
 	}
 	srv := serverutils.StartServerOnly(t, params)
 	defer srv.Stopper().Stop(context.Background())

